util/tlgen/lib.py

```diff
@@ -28,37 +32,37 @@ def simplify_addr(dev: Dict[Any, Any],
     {0x4000_0000, 0x1_0000}, {0x4008_0000, 0x1_0000} then it combines two
     entries into {0x4000_0000, 0x9_0000}
 
-    @param addrs List of Dict[Addr] : {'base_addr':,'size_byte':}
+    @param addrs List of Dict[Addr] : {'base_addr':Dict[ASID,Addr],'size_byte':}
```

Comment:
What is this @param documenting? There is no addrs parameter, and Dict[ASID,Addr] seems an incorrect type.

Reply:
Lets drop that comment.